src/js/main.js

```diff
@@ -124,7 +127,7 @@ function Test(suite, name, func) {
 
   var toolbar = document.createElement('core-toolbar');
   toolbar.setAttribute('class', 'test');
-  var title = document.createElement('span');
+  var title = document.createElement('div');
```

Comment:
This results in a larger progress bar and also gives the new manual start buttons some more room.